ebusd-2.1.x/de/vaillant/hcmode.inc

```diff
@@ -1,17 +1,29 @@
 # type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
 # HC Betriebsart,,,,,,,,,,,,,
 *r,,,,,,"B504",,,,,,,
+*w,,,,,,"B505",,,,,,,
 r,,DateTime,Datum Uhrzeit,,,,00,,,dcfstate;btime;bdate;temp2,,,
+r,,Mode,Betriebsart/Temperatur,,,,01,,,temp0:tempdesired;mcmode:operationmode;days:floorpavingdryingday;temp0;mcmode:teleswitchmode;mctype7:mixertype;IGN;daynight,,,
+r,,Params,TempDesired/TempDesiredLow/HeatingCurve/Type/ShutdownTemp/?/FlowTempMin/FlowTempMax/MaxPreHeating,,,,09,,,temp0:tempdesired;temps:tempdesiredlow;curve;mctype7:mixertype;temp0:shutdowntemp;minutes0:pumpidleperiod;temp0:flowtempmin;temp0:flowtempmax;hours:maxpreheating,,,
+r,,Status,Soll Vorlauf/Status/Ist/?,,,,0D,,,temp0:flowtempdesired;onoff:status;temp:flowtemp;IGN,,,
+r,,Status0a,Status Vorlauf Ist/Mixer/Pumpe/Vorlauf Soll,,,,0A,,,temp;mixer;onoff;onoff;temp0
 r,,Status16,Aussentemperatur,,,,16,,,temp,,,
+w,,SetTempDesired,Solltemperatur setzen,,,,01,,,temp0,,,
```

Comment:
many of those new items don't work on my EHP, so I guess these would have to go to a new hcmode2.inc or so and then included additionally from the circuits supporting it